index.html

```diff
@@ -82,7 +82,7 @@ <h2 style="word-break: break-all" class="to-animate">
                 <a href="./clients.html" class="">CLIENTS</a>
                 <a href="https://opencollective.com/dltxio" target="_blank">OPEN SOURCE</a>
                 <a href="./agile.html">AGILE</a>
-                <a href="./themesh.html">THE MESH</a>
+                <a href="the/mesh.html">THE MESH</a>
```

Comment:
Initial directory context "." has been left out here. Better to keep this consistent. I like the "the" though :-)

Reply:
Webstorm did it, but good catch.